Assets/HoloToolkit/Build/Editor/HoloToolkitCommands.cs

```diff
@@ -1,4 +1,9 @@
-﻿using HoloToolkit.Unity;
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
As @riverar pointed out these files are all MIT licensed. Since this repository is under the Microsoft maintainers we have this branding.